controllers/yawol-controller/loadbalancerset/loadbalancerset_controller.go

```diff
@@ -321,6 +373,9 @@ func (r *LoadBalancerSetReconciler) createMachine(ctx context.Context, set *yawo
 				},
 			},
 			Labels: machineLabels,
+			Annotations: map[string]string{
+				helper.RevisionAnnotation: revision,
```

Comment:
We dont use the var on a other place so I would use it directly
```suggestion
				helper.RevisionAnnotation: set.Annotations[helper.RevisionAnnotation],
```